src/javascript/app/pages/user/account/settings/personal_details.js

```diff
@@ -143,6 +221,10 @@ const PersonalDetails = (() => {
                         if (force_update) { // Force pushing values, used for (API-)expected values
                             $element.attr({ 'data-force': true, 'data-value': el_value });
                         }
+                        // Update data-value on change for inputs
+                        if (should_update_value) {
+                            $(element_key).change(function () {this.setAttribute('data-value', this.value);});
```

Comment:
why is this needed? doesn't existing behaviour handle change of value?

Reply:
Unfortunately no, `data-value` does not get updated when changing data (value does) but `form.handleSubmit` does not pick it up.

Reply:
should we add that in a bigger scale rather than only fixing it in this file then?